frontend/src/components/common/LogViewer.tsx

```diff
@@ -228,10 +229,7 @@ function enableCopyPasteInXterm(xterm: XTerminal) {
         return false;
       }
     }
-    if (arg.ctrlKey && arg.code === 'KeyV' && arg.type === 'keydown') {
-      return false;
-    }
-    return true;
+    return !(arg.ctrlKey && arg.code === 'KeyV' && arg.type === 'keydown');
```

Comment:
This change shouldn't be part of this commit since it's unrelated. If you think it's an important change (I think it's more readable as we have it), then you can propose as a different commit (we try to have atomic commits because of their benefits).

@veyselsahin  I remember you mentioned something about git. If you need help understanding how to extract these changes: create a new branch (just in case) from this one, and then you can do `git reset HEAD^ -p`, which will prompt you for every subset of changes and ask what you want to do; reply with `y` to these ones, and then press `q` to quit.
After that you can commit and only the changes without the one you responded to reset are added: `git commit -amend`.
You will have these copy&paste changes now hanging in there, so you can discard them (`git stash`) or propose them as a new commit (`git add -p` + `git commit`).
After that you can force push to this branch.

After you add fix this 1st commit and before any other commits, you can also fix the failing CI by generating the frontend snapshots (from tests). This can be done by running `make frontend-test` and then answering `a` when it stops the 1st time; after that it should regenerate the snapshots and fail, asking if you want to "update" them, so press `u` and they will be updated.
After this add the new snapshots with `git add -p` and squash them with the previous commit because that's the one that changed the UI (`git commit --amend`).

Reply:
For this specific simplification, feel free to open another issue and PR to close it. I would undo this change in this commit for now and it should be good to review then

Reply:
Dear friends, I've been busy fixing this straightforward issue, but I think it's in the desired state now. I appreciate your patience.